scripts/js/lib/links/ExternalLink.test.ts

```diff
@@ -20,21 +20,38 @@ test("ExternalLink constructor ignores anchors", () => {
 });
 
 test.describe("ExternalLink.check()", () => {
+  let originalFetch: typeof global.fetch;
```

Comment:
Note that this approach would not be safe if Playwright ran the tests inside a file in parallel. However, it runs them sequentially.